recipes/metanetx-sdk/meta.yaml

```diff
@@ -19,19 +19,19 @@ build:
 requirements:
   host:
     - pip
-    - python =3.8
+    - python >=3.8
     - setuptools
   run:
-    - aioftp >=0.13,<1
-    - click >=7.0,<8
-    - click-log >=0.3,<1
-    - depinfo >=1.5,<2
-    - pandas >=0.25,<1
-    - pydantic >=1.4,<2
-    - python =3.8
-    - python-dateutil >=2.8,<3
-    - pytz >=2019.3
-    - toml >=0.10,<1
+    - pin_compatible('aioftp')
+    - pin_compatible('click')
```

Comment:
Do you really need compatible pinnings on things like `click`?

Reply:
I thought I'd go with a safe approach overall. Click broke a few things for me going from version 6 to 7.  So in the hope of making the recipes future proof I was going for the range pins. I can also remove all pins and just hope for the best.

Reply:
Right. The fewer pins, the better in general. up to you!